OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -45,7 +45,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Release</OutputPath>
```

Comment:
It was intentional.
Sorry, I do not have time currently to fix the `make.ps1` script.